src/sql/base/browser/ui/table/formatters.ts

```diff
@@ -122,7 +122,13 @@ export function textFormatter(row: number | undefined, cell: any | undefined, va
 		titleValue = valueToDisplay;
 	}
 
-	return `<span title="${titleValue}" style="${cellStyle}" class="${cellClasses}">${valueToDisplay}</span>`;
+	const formattedValue = `<span title="${titleValue}" style="${cellStyle}" class="${cellClasses}">${valueToDisplay}</span>`;
```

Comment:
That way we'll only have background color for the cell value container instead of the grid cell it cell. Is this what we want?
![image](https://user-images.githubusercontent.com/21186993/226459127-8f9532fd-52df-4276-910e-eb4633346698.png)

Reply:
I see.